core/src/avm2/globals/flash/system/application_domain.rs

```diff
@@ -52,6 +52,9 @@ pub fn get_parent_domain<'gc>(
 ) -> Result<Value<'gc>, Error<'gc>> {
     if let Some(appdomain) = this.and_then(|this| this.as_application_domain()) {
         if let Some(parent_domain) = appdomain.parent_domain() {
+            if parent_domain.is_playerglobals_domain(activation) {
+                return Ok(Value::Null);
```

Comment:
Is there a test for this specific part?

Reply:
We have tests for `stage.loaderInfo.applicationDomain`,but they're unaffected by this PR (since we previously had no parent, and we how have a hidden parent)